tasks/run-file-updates/run-file-updates.yaml

```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: run-file-updates
   labels:
-    app.kubernetes.io/version: "0.4.1"
+    app.kubernetes.io/version: "0.4.2"
```

Comment:
Shouldn't this rather be treated as part of the api changes feature and thus "0.5.0"? I'd say we're not fully done with all the changes yet, so I wouldn't call this a bug. But I don't mind either way.

Reply:
Changed to 0.5.0

Reply:
OK, thanks. We could also change the `fix:` prefix to `feat:`. But I'll leave it up to you.